enterprise_access/apps/bffs/tests/test_response_builders.py

```diff
@@ -0,0 +1,143 @@
+import ddt
+from django.test import RequestFactory, TestCase
+
+from enterprise_access.apps.api_client.tests.test_license_manager_client import TestLicenseManagerUserApiClient
+from enterprise_access.apps.bffs.context import HandlerContext
+from enterprise_access.apps.bffs.response_builder import BaseLearnerResponseBuilder, BaseResponseBuilder
+from enterprise_access.apps.bffs.tests.test_handlers import TestBaseHandlerMixin
+
+
+@ddt.ddt
+class TestBaseResponseBuilder(TestBaseHandlerMixin):
+    def setUp(self):
+        super().setUp()
+        self.base_response_builder = BaseResponseBuilder(self.context)
```

Comment:
[inform] Note: In https://github.com/openedx/enterprise-access/pull/586, now that `HandlerContext` is making API requests during initialization, I ended up having to instantiate context/handlers outside of `setUp` to apply the mock via `@mock.patch`. I'm assuming a similar issue might happen here.

In the linked PR, I ended up removing `TestBaseHandlerMixin` in favor of `TestHandlerContextMixin` re-used as a test utility across the test suites for `HandlerContext` and the handlers. Do you think it makes sense to get that linked PR merged in first or land this PR first, given the likelihood of merge conflicts around these the test mixin?

Also, food for thought: I wonder at what point in the handlers/response we should be relying on a mocked `HandlerContext` so tests for the response builder, etc. don't necessary need to mock stuff related to the context (focusing more on the response builder logic instead). Curious for your two cents here, as well 😄